pkg/console/starter/starter.go

```diff
@@ -176,6 +177,20 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	staleConditionsController := staleconditions.NewRemoveStaleConditions(
+		[]string{
+			// in 4.1.0 we directly set this set of conditions. We should no longer do this.
+			// in 4.3.0+ we can remove this
+			"Degraded",
+			// in follow-up PRs, we should stop using the rest directly:
+			"Progressing",
```

Comment:
Adding the raw `Progressing` here.
If we ever remove any of the new conditions we added, we need to update this controller as well so an operator doesn't get stuck in an upgrade situation.